uni/lib/controller/parsers/parser_course_unit_info.dart

```diff
@@ -42,6 +47,53 @@ Future<List<CourseUnitFileDirectory>> parseFiles(
   return dirs;
 }
 
+Future<Sheet> parseSheet(http.Response response) async {
+  final json = jsonDecode(response.body) as Map<String, dynamic>;
+  final professors = getCourseUnitProfessors(json['ds'] as List<dynamic>);
+  final regents = <Professor>[];
+  final books = <Book>[];
+
+  json['responsabilidades'].forEach((dynamic element) {
+    regents.add(Professor.fromJson(element as Map<String, dynamic>));
+  });
+
+  json['bibliografia'].forEach((dynamic element) {
+    books.add(Book(
+      title: element['titulo'].toString(),
+      isbn: element['isbn'].toString(),
+    ));
+  });
```

Comment:
This would be cleaner with .map() function

example:
`final regents = json["responsabilidades"].map();`